docs/2.7.0/docs/daml/intro/2_DamlScript.rst

```diff
@@ -39,21 +39,19 @@ Before you can create any ``Token`` contracts, you need some parties on the test
 
 - Use of ``<-`` instead of ``=``.
 
-  The reason for that is ``allocateParty`` is an ``Action`` that can only be performed once the ``Script`` is run in the context of a ledger. ``<-`` means "run the action and bind the result". It can only be run in that context because, depending on the ledger state the script is running on, ``allocateParty`` will either give you back a party with the name you specified or append a suffix to that name if such a party has already been allocated.
-
-  More on ``Actions`` and ``do`` blocks in :doc:`5_Restrictions`.
+  The reason for this is that ``allocateParty`` is an ``Action`` that can only be performed once the ``Script`` is run in the context of a ledger. ``<-`` means "run the action and bind the result". It can only be run in that context because, depending on the ledger state, ``allocateParty`` will gives you back a party with the name you specified or appends a suffix to that name if such a party has already been allocated. You can read more about ``Actions`` and ``do`` blocks in :doc:`5_Restrictions`.
```

Comment:
```suggestion
  The reason for this is that ``allocateParty`` is an ``Action`` that can only be performed once the ``Script`` is run in the context of a ledger. ``<-`` means "run the action and bind the result". It can only be run in that context because, depending on the ledger state, ``allocateParty`` gives you back a party with the name you specified or appends a suffix to that name if such a party has already been allocated. You can read more about ``Actions`` and ``do`` blocks in :doc:`5_Restrictions`.
```